test/BUILD

```diff
@@ -63,7 +63,7 @@ scala_binary(
 scala_library(
     name = "HelloLib",
     srcs = ["HelloLib.scala"],
-    plugins = ["@org_psywerx_hairyfotr__linter//jar"],
+    plugins = ["@maven//:org_psywerx_hairyfotr_linter_2_11"],
```

Comment:
I'm still bummed you all didn't find an injective naming scheme. This can collide if you have `org.bar:foo.baz` and `org.bar.foo:baz`. That's a shame. The extra `_` was to separate the org from the group.

Next, since you all are baking the scala version `2_11` into the targets people use, this will mean upgrading a repo will be a pretty major pain. It may be that every string ending in `_2_11"` can be converted to `_2_12"` but previously, we wouldn't have that giant diff. This is a regression in usability for scala users.